lib/github_advisory_sync.rb

```diff
@@ -350,7 +350,10 @@ def create(package)
       Dir.mkdir dir_to_write unless Dir.exist?(dir_to_write)
       File.open(filename_to_write, "w") do |file|
         # create an automatically generated advisory yaml file
-        file.write new_data.to_yaml
+        file.write new_data.merge(
+          { "patched_versions" => vulnerabilities },
+          { "related:\nurl:"  => advisory["references"] }
```

Comment:
I think that should be `{"related" => {"url" => ...}}`.

Reply:
fuxed